prot.c

```diff
@@ -1444,6 +1455,47 @@ dispatch_cmd(Conn *c)
         process_queue();
         return;
 
+    case OP_RESERVE_JOB:
+        if (read_u64(&id, c->cmd + CMD_RESERVE_JOB_LEN, NULL)) {
+            reply_msg(c, MSG_BAD_FORMAT);
+            return;
+        }
+        op_ct[type]++;
+
+        // This command could produce "deadline soon" if
+        // the connection has a reservation about to expire.
+        // But we choose not to do it, until a good reason comes up.
```

Comment:
I agree with this decision, and I'd argue it should be made permanent.

Deadline-soon is necessary for reserve and reserve-with-timeout because they block for an arbitrary amount of time. All other commands, including reserve-job, always complete promptly.

Maybe this comment should be changed to be a description of why there's no deadline-soon response for this command?

Reply:
Done. Also I have created an entry in protocol.txt.